ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/update_available/fixture/UpdateInfoFixture.kt

```diff
@@ -0,0 +1,46 @@
+@file:Suppress("PackageNaming")
+package co.electriccoin.zcash.ui.screen.update_available.fixture
+
+import co.electriccoin.zcash.ui.screen.update_available.AppUpdateChecker
+import co.electriccoin.zcash.ui.screen.update_available.model.UpdateInfo
+import co.electriccoin.zcash.ui.screen.update_available.model.UpdateState
+import com.google.android.play.core.appupdate.AppUpdateInfo
+import com.google.android.play.core.install.model.InstallStatus
+import com.google.android.play.core.install.model.UpdateAvailability
+
+object UpdateInfoFixture {
+
+    val INITIAL_PRIORITY = AppUpdateChecker.Priority.LOW
+    val INITIAL_STATE = UpdateState.Prepared
+    const val INITIAL_FORCE = false
+
+    // just for test purposes we use very simple AppUpdateInfo object instance
+    val APP_UPDATE_INFO: AppUpdateInfo = AppUpdateInfo.zzb(
```

Comment:
This might break between updates to Play Core, because it looks like this is probably a randomized obfuscated name.  So we should think whether there might be some alternatives we can use here.

Reply:
I'm aware of it, thanks for reminding it to me. So imho we have these possible solutions:

1. Change implementation to work with nullable AppUpdateInfo? in the whole app. I had this solution previously, but then changed it.
2. Use some mocking framework, which could build AppUpdateInfo for us.
3. Or I just found `FakeAppUpdateManager` class inside the Play Core lib, which is probabvly desined for these use-cases - [documentation](https://developer.android.com/reference/com/google/android/play/core/appupdate/testing/FakeAppUpdateManager).

What do you think?

Reply:
It seems like `FakeAppUpdateManager` is a good way to go.  The next question is whether we use FakeAppUpdateManager simply to emit an AppUpdateInfo within the code that exists, or whether we leverage it a bit more deeply (eliminating the `val APP_UPDATE_INFO` and refactoring things around a bit).

Reply:
Yep, we rather go to refactoring it a bit. I'll come up with update commit soon.